components/admin/contentPage/Testimonials.tsx

```diff
@@ -23,6 +30,59 @@ const VisuallyHiddenInput = styled("input")({
 });
 
 const Testimonials: React.FC = () => {
+  const [formData, setFormData] = useState<TestsomonialFormData>({
+    profileImage: [],
+    name: '',
+    description: '',
+  });
+  const [Errors, setErrors] = useState<Record<string, string[]> | null>(null);
+
+  // Ensure file-related code only runs in the browser
+  useEffect(() => {
+    if (typeof window === "undefined") {
+      console.error("Running in a non-browser environment.");
+    } else if (typeof File === "undefined") {
+      console.error("File API is not supported in this browser.");
+    }
+  }, []);
+
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.files) {
+      const filesArray = Array.from(event.target.files); // Convert FileList to File[]
+      setFormData((prev) => ({
+        ...prev,
+        profileImage: filesArray,
+      }));
+    }
+  };
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+    const { name, value } = event.target;
+    setFormData((prev) => ({
+      ...prev,
+      [name]: value,
+    }));
+  };
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault(); 
+  const { errors: validationErrors, data } = validateFormData(schemaAdminPanelTestimonials, formData);
```

Comment:
This code is likely to give errors as the common validation function was updated. So handle validation errors as a single concatenated string per field. Refer the handle submit function in LogoCarousel.tsx (content page -  Logo Carousel).